Lesson6.1/src/by/teachmeskills/robot/Robot.java

```diff
@@ -0,0 +1,53 @@
+package by.teachmeskills.robot;
+
+import by.teachmeskills.robot.interfaces.IRobot;
+import by.teachmeskills.robot.interfaces.IRobotPart;
+
+public class Robot implements IRobot {
+    private IRobotPart head;
+    private IRobotPart hand;
+    private IRobotPart leg;
+
+    public Robot(IRobotPart head, IRobotPart hand, IRobotPart leg) {
```

Comment:
writing it like this would allow to create a robot using 3 hands. I don't think that this is a good practice

Reply:
if you want to leave it as it is, you need to make robot fabric the only place that can create robots